csrankings-h.csv

```diff
@@ -919,3 +919,5 @@ Hyunsoo Yoon,KAIST,http://nslab.kaist.ac.kr/members_professor.htm,bmGTi6YAAAAJ
 Hyunsook Do,University of North Texas,http://www.cse.unt.edu/~hdo,45vFd_0AAAAJ
 Hywel T. P. Williams,University of Exeter,http://emps.exeter.ac.uk/computer-science/staff/htpw201,vARIotQAAAAJ
 Hywel Thomas Parker Williams,University of Exeter,http://emps.exeter.ac.uk/computer-science/staff/htpw201,vARIotQAAAAJ
+Hongyuan Zha,CUHK-SZ,https://sds.cuhk.edu.cn/teacher/65,n1DQMIsAAAAJ
+Haizhou Li,CUHK-SZ,https://colips.org/~eleliha/,z8_x7C8AAAAJ
```

Comment:
Hi, thanks for your help. So for Prof. Haizhou Li, I should replace his name with Haizhou Li 0001, which is the name on DBLP? I have just followed the instruction and removed the suffix since it says that please do not use the numeric suffixes.

Reply:
In fact it says the opposite ("include disambiguation suffixes like 0001 as needed"); the suffixes referred to a prior version of CSrankings' data files, which ended in 0-9. (I will remove that text to avoid further confusion!).

Reply:
Got it. Thanks!